site/content/docs/manifest/scheduled-job.en.md

```diff
@@ -22,6 +22,11 @@ List of all available properties for a `'Scheduled Job'` manifest. To learn abou
       LOG_LEVEL: info
     secrets:
       GITHUB_TOKEN: GITHUB_TOKEN
+      
+    # You can override any of the values defined above by environment.
+    environments:
+      prod:
+        cpu: 2048               # Larger CPU value for prod environment 
```

Comment:
nit: for this they will also need to scale the memory as well! 
If we can update to include a mem field as well 🙏

Reply:
Done. 
Should add a `memory` field to the commented out example in the manifest template? Or link to https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html?

Reply:
I think linking to it in our [`cpu`/`memory`](https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#cpu) docs is sufficient